joomla-projects/privacy-framework
# Display correct actions log even plugins disabled

Pull Request for Issue #126.
### Summary of Changes
This PR adds a method to load all actionlog plugins language file and calls it each time actions are being displayed to make sure the actions are displayed properly even if the plugins are disabled

### Testing Instructions
1. See https://github.com/joomla-projects/privacy-framework/issues/126 for issue description
2. Apply patch, confirm the issue is sorted.
